homeassistant/components/renson/sensor.py

```diff
@@ -292,6 +290,10 @@ def _handle_coordinator_update(self) -> None:
 
         if self.raw_format:
             self._attr_native_value = value
+        elif self.entity_description.device_class == SensorDeviceClass.ENUM:
+            self._attr_native_value = OPTIONS_MAPPING[
```

Comment:
Would it be safer to use `get()` here?

Reply:
Markted the PR as draft for now

Reply:
Oh did not see this comment. Will check it later today!